var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -62,12 +66,26 @@ def install(self, spec, prefix):
                 '--with-hwloc=%s' % spec['hwloc'].prefix,
                 '--with-serial'
             ]
+            arch_args = []
             if '+openmp' in spec:
                 g_args.append('--with-openmp')
             if 'qthreads' in spec:
                 g_args.append('--with-qthreads=%s' % spec['qthreads'].prefix)
             if 'cuda' in spec:
                 g_args.append('--with-cuda=%s' % spec['cuda'].prefix)
+            # 1)  hopefully we have a host architecture
```

Comment:
I don't really like "hopefully"s. We should check this is the case with a `conflicts` directive.

Can `--arch` accept multiple architectures? We should use a multi-valued variant here.

Reply:
You are right, to enable `Pascal60` one of them need to be on.

Reply:
Thank you @adamjstewart and @junghans. A suggested template was Charm which I will reference.

Reply:
Even if not a multi-valued variant, it should be a string-type variant.